.gitignore

```diff
@@ -80,13 +80,13 @@ lib-cov
 # Coverage directory used by tools like istanbul
 coverage
 
-# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+# Grunt intermediate storage (https://gruntjs.com/creating-plugins#storing-task-files)
```

Comment:
Is that okay ? You created that gitignore with gitignore.io, it is basically a preset which handle multiple developer using different ide, tools or such